lib/chart/animated_line_chart.dart

```diff
@@ -242,7 +242,7 @@ class ChartPainter extends CustomPainter {
       TextSpan span = TextSpan(
           style: TextStyle(
               color: _chart.lines[index].color,
-              fontWeight: FontWeight.w200,
```

Comment:
Could we expose this field so it is configurable from users of the plugins?

Reply:
exposed the tap text fontweight